recipes/notcurses/all/conanfile.py

```diff
@@ -0,0 +1,41 @@
+from conans import ConanFile, CMake, tools
+
+
+class NotcursesConan(ConanFile):
+    name = "notcurses"
+    description = "a blingful TUI/character graphics library"
+    topics = ("graphics", "curses", "tui", "console", "ncurses")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://nick-black.com/dankwiki/index.php/Notcurses"
+    license = "Apache-2.0"
+    exports_sources = ["CMakeLists.txt"]
+    settings = "os", "compiler", "build_type", "arch"
+    requires = "openimageio/2.2.7.0", "libunistring/0.9.10"
```

Comment:
```suggestion
```

Reply:
I don't understand. If I remove this line, I don't complete `conan create`, and I don't understand how anything would know about the dependencies?

Reply:
I've changed it to a `requirements(self)` method, ala [this example](https://github.com/conan-io/conan-center-index/blob/3a29c482525263b51d91831b5ded0de1065807af/recipes/mongo-c-driver/all/conanfile.py). Is that what you meant?

Reply:
Ahh, I see it is from your text above. Done!